packages/samples/react/src/components/table/column-alignment.tsx

```diff
@@ -72,7 +72,6 @@ export const TableColumnAlignment: FC = () => (
 				_headers={{
 					horizontal: [
 						[
-							{ label: '', asTd: true },
```

Comment:
@deleonio Ich möchte nur noch einmal darauf eingehen, danach lasse ich das Thema gerne ruhen:

Bewirkt die Notwendigkeit, diese Platzhalter-Zellen zu entfernen, nicht einen Breaking Change in allen Anwendungen, die bereits komplexe Tabellen verwenden?